src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -1759,6 +1759,38 @@ int32_t SurgeGUIEditor::controlModifierClicked(CControl* control, CButtonState b
          sprintf(txt2, "Value: %s", txt);
          contextMenu->addEntry(txt2, eid++);
          bool cancellearn = false;
+         int ccid = 0;
+          bool handled = false;
+
+            ccid = modsource - ms_ctrl1;
+            contextMenu->addEntry("-", eid++);
+            // Construct submenus for explicit controller mapping
+             COptionMenu *midiSub = new COptionMenu(menuRect, 0, 0, 0, 0, VSTGUI::COptionMenu::kNoDrawStyle);
+             COptionMenu *currentSub;
+             for( int mc = 0; mc < 128; ++mc )
+             {
+                 if( mc % 20 == 0 )
+                 {
+                     currentSub = new COptionMenu( menuRect, 0, 0, 0, 0, VSTGUI::COptionMenu::kNoDrawStyle );
+                     char name[ 256 ];
```

Comment:
`char name[256]`

Reply:
changed `name[ 256 ];` to `name[256];`. that's right, right?